dspace/config/registries/notify-types.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE dspace-dc-types SYSTEM "dspace-dc-types.dtd">
+
+<dspace-dc-types>
+    <dspace-header>
+        <title>Notify fields definition</title>
+    </dspace-header>
+    
+    <dc-schema>
+        <name>notify</name>
+        <namespace>http://dspace.org/notify</namespace>
```

Comment:
Why do we have two new schemas in this PR?  One is "coar.notify.*" and one is "notify.*".  Won't this get confusing as to which one to use?  Is there a reason why we don't just combine these into one?